testing/integ-test/build.gradle.kts

```diff
@@ -26,7 +26,6 @@ dependencies {
 
     integTestImplementation(libs.samplesCheck) {
         exclude(group = "org.codehaus.groovy", module = "groovy-all")
```

Comment:
Doesn't necessarily need to be done here, but we should probably replace this exclude with component metadata rules as well, as I see at least 3 places in this pr diff alone that add the exclude